### ISSUE TITLE
Sub-View Back button bug

### ISSUE THREAD

**Issue by kanichrnd** [Clooos/Bubble-Card#590]
**Describe the bug**  
If you use bubble card in Sub-View page, and then want to go back to main view page, you have to press back button as many times as you opened the bubble card.

**To Reproduce**  
1. Create Sub-View page
2. Create bubble card inside it
3. Open and close bubble card 3-5 times
4. Try to go back from Sub-View by pressing Back button

**Expected behavior**  
Back button should work from the 1st click

**Informations (please complete the following information):**  
 - OS: Windows and iOS
 - Edge, Firefox
 - 2.0.4
 - 2024.6.3

Thank you! 🍻

**Comment by Mofeywalker** [comment]
I have the same problem, but more specific. It's only when a popup is used un a sub view.

**Comment by Clooos** [comment]
Hi, I've changed something in the latest release that might have fixed that, can I have some feedback? 🤞

https://github.com/Clooos/Bubble-Card/releases/tag/v2.2.4

**Comment by Mofeywalker** [comment]
Hi @Clooos unfortunately, the popup is now opened again when I hit the back button in the sub view. Now it is not possible to go back to the main view.

I attached a little screen recording which shows the behaviour.

https://github.com/user-attachments/assets/b69edf3d-2faf-4c5b-af05-d41834e4625b

Thank you again for your time and effort!

**Comment by Clooos** [comment]
Thank you for the details! I have misunderstood this issue and this shouldn't be too difficult to fix.

**Comment by Mofeywalker** [comment]
Hi, I found a way to fix this behaviour. There is an option `back_path: /lovelace/home` which can be applied on the subviews. I found the solution in this [issue](https://github.com/home-assistant/frontend/issues/15424#issuecomment-1615886905). This allows to always return back to the home view instead of "back button" press. :)

**Comment by himisk2022** [comment]
That us a good solution! unfortunately not if you open popups on the main page and want to close the app with the back button or want to return to the normal history in the browser.

**Comment by lennon101** [comment]
+1 on this issue. If I use a bubble pop up card in a sub view, I now can't navigate back to the main view because the back button of the sub view returns me to the bubble pop up card instead (as shown in the screen recording) 

https://github.com/user-attachments/assets/edaaeded-8873-4b93-8faa-9dc61793d08e

**Comment by Clooos** [comment]
There is sadly nothing that I can do directly in Bubble Card for this, that button acts as a back button and when you open a pop-up the URL is modified, so pressing that button will automatically re-open that pop-up.

But have you checked this comment? This should fix your issue in that specific use case.

https://github.com/Clooos/Bubble-Card/issues/590#issuecomment-2352996076

**Comment by lennon101** [comment]
Yeah leagend! I did indeed see that comment but I didn't read it in it's completeness and thought the fix wasn't going to apply in my case. But I just applied it and it does indeed resolve my issue. I appreciate that (due to the nature of how the bubble card works regarding modifying the url and therefore the history of the urls) there isn't much that can be done from here. I recommend this issue be closed for now with a comment pinned regarding the fix (for those where the fix applies 😄) 

Or perhaps the fix could be documented in a FAQ or a note in the ReadMe regarding sub-views and the back button? 

Cheers, and again: amazing work!  🎉

**Comment by Clooos** [comment]
Good idea indeed! I will let this issue open until I've documented this fix 👌

**Comment by MrBearPresident** [comment]
@Knight1981 Merging from #1069.

**Comment by HyperCriSiS** [comment]
> Hi [@Clooos](https://github.com/Clooos) unfortunately, the popup is now opened again when I hit the back button in the sub view. Now it is not possible to go back to the main view.
> 
> I attached a little screen recording which shows the behaviour.
> Screen_Recording_20240824_090034_Home.Assistant.mp4
> 
> Thank you again for your time and effort!

What icon pack have u installed?
Also having a shield and was searching for a suitable icon 😃

**Comment by Mofeywalker** [comment]
> > Hi [@Clooos](https://github.com/Clooos) unfortunately, the popup is now opened again when I hit the back button in the sub view. Now it is not possible to go back to the main view.
> > 
> > I attached a little screen recording which shows the behaviour.
> > Screen_Recording_20240824_090034_Home.Assistant.mp4
> > 
> > Thank you again for your time and effort!
> 
> What icon pack have u installed?
> Also having a shield and was searching for a suitable icon 😃

Hi, I use the following icon Pack :)

https://github.com/elax46/custom-brand-icons
